.github/workflows/basic.yml

```diff
@@ -73,49 +65,35 @@ jobs:
         if: github.event.schedule=='0 4 * * 0,1,2,3,4'
         run: |
           echo "NETWORK=devnet" >> $GITHUB_ENV
-      - name: Setup env
-        id: setup
-        run: |
-          # $1 - inputs
-          # $2 - env.VAR
-          function setVar {
-            if [ -z "$1" ]
-              then
-                RESULT="$2"
-              else
-                RESULT="$1"
-            fi
-            echo $RESULT
-          }
-
-          NETWORK=$( setVar "${{ github.event.inputs.network }}" "${{ env.NETWORK }}" )
-          RUNNER=$( setVar "${{ github.event.inputs.runner }}" "${{ env.RUNNER }}" )
-          NUMPROCESSES=$( setVar "${{ github.event.inputs.numprocesses }}" "2" )
-
-          echo "Network: ${NETWORK}"
-          echo "Runner: ${RUNNER}"
-          echo "Numprocesses: ${NUMPROCESSES}"
-          echo "Build url: ${{ env.BUILD_URL }}"
-
-          echo "network=${NETWORK}" >> $GITHUB_OUTPUT
-          echo "runner=${RUNNER}" >> $GITHUB_OUTPUT
-          echo "jobs=${JOBS_NUMBER}" >> $GITHUB_OUTPUT
-          echo "numprocesses=${NUMPROCESSES}" >> $GITHUB_OUTPUT
+      - uses: actions/checkout@v4
```

Comment:
If we want run our tests on night-stand or devnet why we need to setup terraform?

Reply:
We have already disscussed it
1) it’s down very often
2) it’s updaiting only 1 time a day
3) The configuration is not the same as in proxy CI
4) we can't run tests on several versions (for now we have changes in 1.9.x and in develop)
5) we can't run operator tests in the middle of day on night stand (we can't be sure that nobody use it)

night-stand and devnet are good for night runs ,  not for runs for each PR